web/src/buddy_list.ts

```diff
@@ -496,7 +496,7 @@ export class BuddyList extends BuddyListConf {
             $(
                 render_section_header({
                     id: "buddy-list-other-users-section-heading",
-                    header_text: $t({defaultMessage: "Others"}),
+                    header_text: $t({defaultMessage: "OTHERS"}),
```

Comment:
I'm not sure if we need to capitalize these strings or should use `text-transform` instead. I guess this is probably handier for translators?

Reply:
I was mostly just copying what I saw in `left_sidebar.hbs`:

```
<h4 class="left-sidebar-title"><span class="streams-tooltip-target" data-tooltip-template-id="filter-streams-tooltip-template">{{t 'CHANNELS' }}</span></h4>
```

Reply:
Yeah this is probably better; it's just a bit annoying for translators to need to retranslate the strings. Still, clearer about how they are used.